### PR TITLE
receptor specific plots

### PR DESCRIPTION
**Pull request by ajtsao1**
@aarmey I added the receptor-specific phosphorylation plots to the compartmental model plotting notebook. I can work on adding a more concise function for these into the plots.jl file as well. 
I think my last PR on this notebook had a merge block but the changes are reflected here, so I will close the other one.

### CODE REVIEW

**Comment by aarmey** [comment]
Yes, if this subsumes the other PR then close it and delete the branch.

I'm surprised there's essentially no difference. What concentration of Gas6 is this stimulating with? I wonder if a much lower amount shows different behavior...

**Comment by ajtsao1** [comment]
@aarmey Got it, deleted the branch and closed the PR.
I reran the notebook using the 200th index for the parameters, after Sumedha pointed out that the parameters reach steadier values at the later values. So gasCur is around 0.12 and the plots looked the same as these ones.
I can try out some smaller values and see if there are significant changes.

**Comment by ajtsao1** [comment]
@aarmey I think my new commit has been added to this PR. 
Now this notebook has the total receptor pY, dimers, and receptor-specific pY plots. I’m not sure if this was correct, but I used the recpSpecific vectors to come up with the dimer vectors. The dimers are being plotted at timepoint 1 (0.01 sec). I also changed the diffusivity from 1 to 0.1 but the spatial patterns remained the same.

File: CompPlots.ipynb [hunk 1]
```diff
@@ -23,15 +14,137 @@
...
+    "MerTK = 55\n",
+    "Tyro3 = 3.7\n",
+    "\n",
+    "CompParams = vcat(dif, gasPart, gasCur, xFwd[1], AXL, MerTK, Tyro3, Ig2rev[1], [1.0, 1.0, 1.8, 100.0]);\n",
+    "\n",
+    "tps = @SVector Float64[0.01, 0.1, 1];\n",
+    "r = 1:100;"
+   ]
+  },
+  {
```
Comment (aarmey) [inline CompPlots.ipynb:83]: @ajtsao1 One thing that would be helpful to take care of first is turning all this code into a consistent function. I could see a function where you input the parameters, time points, and the reduction vector, and it produces the plot for you. That would turn much of this notebook into just a few lines.

In the mean time, I'm also digging into the compartmental model a bit more.

**Comment by aarmey** [comment]
Another thing to look at @ajtsao1 is the mechanism of activation happening here. It seems like much of the increase in receptor pY comes from diffusion of the receptor into the PS region. I wonder what it looks like if you normalize the pY by the total receptor. This would help to reveal whether the pY increase is purely a reflection of greater receptor.

**Pull request closed without merging**